dlt/_workspace/helpers/dashboard/dlt_dashboard.py

```diff
@@ -1002,8 +1002,10 @@ def utils_discover_pipelines(
     # sync from runtime if enabled
     _tmp_config = utils.resolve_dashboard_config(None)
     if _tmp_config.sync_from_runtime:
```

Comment:
I don't think we really need this var. If the credentials are available, we should sync. But this is not important rn.